### PR TITLE
Switch mode button

### PR DESCRIPTION
**Pull request by osugraytr**
For now, if you want to see a change take this line:

	  ------> 	<button type="button" class="modeButton_spanish">
				<span>MODE: SPANISH</span>
			</button>

and change "modeButton_english" to "modeButton_spanish"

### CODE REVIEW

**Comment by kevinkoos** [comment]
Switch button looks great. Try giving the button a class all to it's and a class for the specific decktype. That way there is less duplicate code. Now that it's in we might need to find a better place in the bottom bar which is getting crowded.
The button in the modal in the end doesn't appear correctly. We may not need a mode switch button there.

File: src/main/java/models/Game.java [hunk 1]
```diff
@@ -87,6 +87,21 @@ public void reset() {
         this.deck.shuffle();
         this.dealFour();
     }
+    
+    public void switchDeck() {
```
Comment (kevinkoos) [inline src/main/java/models/Game.java]: This is the same code as the reset method. We're looking to change what kind of deck the user's game is using by changing the `game.deckType` variable to the other ('E' and 'S') and then calling `reset()`.

**Comment by thadenf** [comment]
changes look good, form what I understand. I will clone and run with my work in progress.

**Comment by mgray35** [comment]
Switch method implemented to act with button

**Comment by kevinkoos** [comment]
Downloaded branch onto my local machine. Everything checks out. Switch button works as intended,

**Pull request merged**